MozillaReality/FirefoxReality/app/src/main/res/values/strings.xml

```diff
@@ -1318,6 +1326,9 @@
     <!-- This string is shown in the context menu after a longpress on a text. When clicked it unselects the selected text. -->
     <string name="context_menu_unselect">Unselect</string>
 
+    <!-- This string is shown in the context menu after a longpress on a text. When clicked search for the selected text in the active search engine. -->
+    <string name="context_menu_web_search">Search with %1$s</string>
```

Comment:
We'll need to know in the comment what the placeholder will be replaced with here.
CC @keianhzo

Reply:
Thanks Delphine, it's updated here: https://github.com/MozillaReality/FirefoxReality/pull/3631